? 0-views-file_managed_extras.patch
? 563020_views_object_cache_0.patch
? 667950-javascript-states-revert_1.patch
? 667950-javascript-states-revert_1_0.patch
? 667950-javascript-states-revert_1_0_2_0.patch
? 694094-drupal_static.patch
? 741234_views_empty_arg_check_null.patch
? 863478-distinct-support-broken.patch
? 881060.patch
? 884730.patch
? 894618-filter_date-between.patch
? 895046-field_multiple.patch
? 930582-fix.patch
? 931552-fix.patch
? 931886.patch
? 931886_0.patch
? 949526-external-db.patch
? 952636-filter.7.patch
? 960528-tablesort.patch
? 960596-quote.patch
? 963454.patch
? 971326-join_deleted.patch
? 974542-notice-pre_render.patch
? 975400-refactor-field_render.patch
? 975400-refactor-field_render.ptach
? 976098-field-drupal_render-notice.patch
? FALSE,
? b
? d7_views_taxonomy_handlers.patch
? filter_format_4.patch
? foo.patch
? notices.patch
? revert.patch
? sites
? views-field-comment-node-link-call-time-pass-by-reference.patch
? views-filter_numeric_test_0.patch
? views-or.patch
? views-term_fields_contain_tid_only.patch
? views-term_fields_contain_tid_only.patch.1
? views-test-user_argument_default.patch
? views.aggregator_feed_item_id.patch
? views_aggregator_fid_filter.patch
? views_formatter_settings-884730-
? views_formatter_settings-884730-43.patch
? views_formatter_settings-884730-61.patch
? views_formatter_settings-884730-63.patch
? views_formatter_settings.patch
? tests/taxonomy
Index: plugins/views_plugin_query_default.inc
===================================================================
RCS file: /cvs/drupal-contrib/contributions/modules/views/plugins/Attic/views_plugin_query_default.inc,v
retrieving revision 1.1.4.30
diff -u -p -r1.1.4.30 views_plugin_query_default.inc
--- plugins/views_plugin_query_default.inc	5 Nov 2010 13:47:14 -0000	1.1.4.30
+++ plugins/views_plugin_query_default.inc	21 Nov 2010 00:33:54 -0000
@@ -937,7 +937,7 @@ class views_plugin_query_default extends
       $fieldname = (!empty($field['alias']) ? $field['alias'] : $string);
 
       if (!empty($field['distinct'])) {
-        throw new Exception("d7cx: Column-level distinct is not supported yet.");
+        throw new Exception("Column-level distinct is not supported anymore.");
       }
 
       if (!empty($field['count'])) {
@@ -1001,7 +1001,7 @@ class views_plugin_query_default extends
   function query($get_count = FALSE) {
     // Check query distinct value.
     if (empty($this->no_distinct) && $this->distinct && !empty($this->fields)) {
-      $base_field_alias = $this->add_field($this->base_table, $this->base_field, NULL, array('distinct' => TRUE));
+      $base_field_alias = $this->add_field($this->base_table, $this->base_field);
       $this->add_groupby($base_field_alias);
     }
 
@@ -1105,7 +1105,7 @@ class views_plugin_query_default extends
   function build(&$view) {
     // Make the query distinct if the option was set.
     if (!empty($this->options['distinct'])) {
-      // $this->set_distinct();
+      $this->set_distinct();
     }
 
     $view->init_pager();
